### PR TITLE
Add daglig ordination tests

### PR DESCRIPTION
**Pull request by jhviggo**
solves #8

- add DagligFast class tests
- add DagligSkaev class tests

### CODE REVIEW

File: test/DagligSkaevTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import java.time.LocalTime;
+
+import org.junit.Test;
+
+import ordination.DagligSkaev;
+import ordination.Laegemiddel;
+
+public class DagligSkaevTest {
+
+    Laegemiddel lm1 = new Laegemiddel("CBD", 2, 4, 6, "ML");
```
Comment (jhviggo) [inline test/DagligSkaevTest.java:13]:
```suggestion
    Laegemiddel lm1 = new Laegemiddel("CBD", 2, 4, 6, "ml");
```
Small thing: This is capitalized when the others are not

File: test/DagligSkaevTest.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+        assertEquals(1, ds1.getDoser().get(1).getAntal(), 0.001);
+
+        assertEquals(14, ds1.getDoser().get(2).getTid().getHour(), 0.001);
+        assertEquals(15, ds1.getDoser().get(2).getTid().getMinute(), 0.001);
+        assertEquals(2, ds1.getDoser().get(2).getAntal(), 0.001);
+
+    }
+
+    @Test
+    public void samletDosis() {
```
Comment (jhviggo) [inline test/DagligSkaevTest.java:38]: Maybe add more than one test for this method?

**Review by jhviggo — COMMENTED** [review]
Looks fine. I only had a few small comments that you can either fix or ignore.

**Review by hctankred — APPROVED** [review]
Looks fine.

**Pull request merged**